### ISSUE TITLE
src with cache are not update after files changed

### ISSUE THREAD

**Issue by vikdet** [nmrugg/gulp-hash-src#6]
I have one script 
`<script src="Scripts/dest/engineJs.js"></script></code>`
which is changed after plugin to
`<script src="Scripts/dest/engineJs.js?cbh=e9af0a7edcdb8a013fea4af9c045aac2"></script>`
But then when i change engineJs.js file and rerun plugin, script's src isn't changed. I think src should be updated to new hash.

**Comment by nmrugg** [comment]
Sorry for the delay. Do you mean that you are running the script on a file that already has the `?cbh=...` in it?

I suppose that makes sense, but it would be a bit trickier to parse to make find the correct file and not mess up the query string as well. But I suppose that could be done.

**Comment by vikdet** [comment]
Yes, I am running he script on a file that already has the ?cbh=... in it.

I have found that it is already not necessary, because it makes files always changed and tracked by git. This is not convinient approach in dev environment.
